_includes/v22.2/orchestration/kubernetes-upgrade-cluster-helm.md

```diff
@@ -249,6 +249,8 @@
         > RESET CLUSTER SETTING cluster.preserve_downgrade_option;
         ~~~
 
+        After the upgrade to {{ page.version.version }} is finalized, you may notice an increase in compaction activity due to a background migration within the storage engine. To observe the migration's progress, check the **Compactions** section of the [Storage Dashboard](ui-storage-dashboard.html) in the DB Console or monitor the `storage.marked-for-compaction-files` time series metric. When the metric's value nears or reaches `0`, the migration is complete and compaction activity will return to normal levels.
```

Comment:
@jbowens PTAL. Identical language is used in a few separate places in this PR, and also in https://github.com/cockroachdb/docs/pull/15712.

Reply:
lgtm!